### DISCUSSION TITLE
Github Oauth token.

### DISCUSSION THREAD

**Discussion by edutrul** [renovatebot/renovate#18200] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

I get an error about the GitHub token configuration.
To reproduce this error, follow these steps:
1- I made a fork to [renovatebot](https://github.com/renovatebot/renovate)
2- I ran `yarn install`
3-I ran `yarn build`
4-Run `yarn start`.
And this is where this error comes out.
<img width="633" alt="image" src="https://user-images.githubusercontent.com/1582129/194635847-d89d1ec7-2a89-4769-82c6-bd7f034803be.png">

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction, but I have linked to a public repo where it occurs

**Comment by edutrul** [comment]
In the renovate specifications, they do not clarify that if the config.js file does not exist, it should be created.
So to solve this, I had to create a config.js file in my project, where I put the code:
`module.exports = {
  token: 'xxxxxxxx'
};`
And I reran yarn start, and everything worked fine

**Comment by edutrul** [comment]
A pull request will be made about this error

**Comment by adam-moss** [comment, marked as answer]
It isn't an error, see https://docs.renovatebot.com/getting-started/running/#githubcom-token-for-release-notes

You can put the token in an env var

**Reply by rarkins** [reply]
Or CLI
